core/game_events.go

```diff
@@ -9,13 +9,14 @@ type PlayerStarts struct {
 }
 
 type NewFood struct {
+	ID int		 // ID of food
 	Pos Coord    // coordinates of new food
 }
```

Comment:
These comments duplicate what the names and types of the variables say, consider removing them.